apps/amiapp_flutter/android/app/build.gradle

```diff
@@ -40,6 +43,7 @@ android {
         targetSdkVersion 33
         versionCode 1
         versionName "1.0"
+        multiDexEnabled true
```

Comment:
I think that this increases build times when it's enabled so just calling it out to make sure this is required and it wasn't automatically added or you forgot to remove it. If you added it intentionally, no problem.

Reply:
Correct. This too was added only for [flutter_local_notifications gradle setup](https://pub.dev/packages/flutter_local_notifications#gradle-setup).